lib/cc/analyzer/issue.rb

```diff
@@ -39,20 +39,16 @@ def fingerprint
       attr_reader :output
 
       def default_fingerprint
-        digest = Digest::MD5.new
-        digest << path
-        digest << "|"
-        digest << check_name.to_s
-        digest.to_s
+        if ENV["CODECLIMATE_SOURCE_FINGERPRINT"]
+          ::CC::CLI::SourceFingerprint.new(parsed_output).compute
+        else
+          ::CC::CLI::PathFingerprint.new(parsed_output).compute
```

Comment:
I think this is telling you these objects should live under the `Analyzer` namespace. Is there a reason you went with `CLI`?